BetterMD/elements/table.py

```
@@ -14,92 +14,153 @@
+            raise ValueError("THead with multiple rows is not supported when `experiments.multiheader` is `False` as it is not standard markdown")
+
+        rows = [
+            [
+                e.to_md().replace("\n", "<br />") for e in row.data
+            ] for row in symbol.data
+        ]
+
+        def op_get(list:'list', i:'int', d):
+            try:
+                return list[i]
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)